tools/runner/runner.go

```diff
@@ -117,14 +120,13 @@ func (r *Runner) runTest(config *grpcv1.LoadTest, reporter *TestCaseReporter, do
 	for {
 		loadTest, err := r.loadTestGetter.Get(config.Name, metav1.GetOptions{})
 		if err != nil {
-			reporter.Warning("Failed to poll test %s: %v", name, err)
 			if retries < r.retries {
 				retries++
-				reporter.Info("Scheduling retry %d/%d to poll test", retries, r.retries)
+				reporter.Info("Failed to poll test %s (scheduling retry %d/%d): %v", name, retries, r.retries, err)
```

Comment:
It is not necessary to mention the name of the test. I would use comma instead of parentheses.